src/sentry/rules/actions/utils.py

```diff
@@ -131,7 +131,7 @@ def get_changed_data(
             pass
 
         if environment:
-            changed_data["environment"].append(f"Removed *{environment.name}* environment")
+            changed_data["environment"].append(f"Removed '{environment.name}' environment")
```

Comment:
The logic here and in the previous if statement feel incredibly similar.. could this be simplified a bit and then we use a helper function to decide if there's an update required?

for example:
```python
added_env = get_environment_change(rule_data, rule_data_before)
before_env = get_environment_change(rule_data_before, rule_data)

if added_env:
  ...
  
if before_env:
  ...
```
This allows us to simplify the code / remove the duplicate logic. it also would match the way the other if statements work.. (could likely put those into a dictionary and enumerate each change data in a for loop then, drastically simplify this code)

```python
data_to_check = {
   label_text: check_value_changed(rule_data, rule_data_before, "label", "rule name"),
   action_match: check_value_changed(rule_data, rule_data_before, "action_match", "trigger"),
   ...
}

for key, value in data_to_check:
  if value:
     changed_data[key] = value
```